client/src/components/layouts/Default.js

```diff
@@ -157,11 +154,8 @@ class DefaultLayout extends Component {
           <style>{fontawesome.dom.css()}</style>
         </Helmet>
         <WithInstantSearch>
-          <Header disableSettings={landingPage} />
-          <div
-            className={`default-layout
-          ${landingPage ? 'landing-page' : ''}`}
-          >
+          <Header disableSettings={!isSignedIn} />
```

Comment:
It also removes the search bar from the header for not logged in users.

Reply:
Once the roll out the universal nav which would be similar to nav news, I believe there would not be a harm in having the search available all the time

Reply:
then while we have not done this, let's remove this check and will show search bar all the time:

https://github.com/freeCodeCamp/freeCodeCamp/blob/7065ace345f61f1ec628889ec155bc89e29f1a52/client/src/components/Header/index.js#L23

Reply:
That for catching that @ValeriaS.
the pr has been updated accordingly.